gatsby-node.js

```diff
@@ -143,7 +146,7 @@ exports.createPages = ({ actions, graphql }) => {
       pageTemplate: path.resolve(`src/templates/blog.js`),
       paginatePost: "/blog", // old field. not remove
       pathPrefix: "/blog", // new field. not remove
-      pageLength: 8,
+      pageLength: 100,
```

Comment:
It seems too many posts for one page, I would set a max of 20 posts for each page.